src/CSV/ImportResult.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace App\CSV;
+
+use App\Entity\Donation;
+use App\Entity\Person;
+use App\Entity\Project;
+use App\Entity\Reward;
+use App\Interfaces\CSV\ImportResultInterface;
+
+final class ImportResult implements ImportResultInterface
+{
+    public function __construct(
+        private array $persons,
+        private array $projects,
+        private array $donations,
+        private array $rewards,
+        private ?string $errorsString = null
```

Comment:
errors should be an array , of objet like ImportResultError with some attributes like "string $path" and "ConstraintViolationListInterface $violations".